demosymfonyformsimple/composer.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "prestashop/demosymfonyformsimple",
+    "description": "PrestaShop - Settings Form Examples",
+    "license": "AFL-3.0",
+    "authors": [
+      {
+        "name": "PrestaShop Core Team"
+      }
+    ],
+    "autoload": {
+      "psr-4": {
+        "PrestaShop\\Module\\DemoSymfonyFormSimple\\": "src/"
+      }
+    },
+    "require": {
+      "php": ">=7.1.0"
```

```suggestion
      "php": ">=7.2.5"
```

PS8 is compatible with minimum PHP 7.2.5